web/src/styles/components/quick-action-tile.scss

```diff
@@ -19,6 +19,9 @@
 .quick-action-tile {
   background-color: $base-extra-light;
   border: 2px solid $base-lightest;
+  cursor: pointer;
+  height: 100%;
+  line-height: 160%;
```

Comment:
[dust] Should we also disable highlighting too?

Reply:
Great idea. Included.